<head>
  <style>
      div.tooltip-donut {
        position: absolute;
        text-align: center;
        padding: .5rem;
        background: #FFFFFF;
        color: #313639;
        border: 1px solid #313639;
        border-radius: 8px;
        pointer-events: none;
        font-size: 1.3rem;
    }
  </style>
</head>

<body>
	<table width=1100>
		<tr>
			<td>
				<br>
				<h5>GutFeeling KnowledgeBase</h5>
				<br>
				<span style="font-size:13px;">
					We have developed a proof-of-concept gut microbiome monitoring system using a sequencing and analysis pipeline implemented during our previous I-Corps award (see below).
					<br><br>
					We have collected from the individuals enrolled in our study the following: three separate fecal samples for metagenomic sequencing, anthropometric measurements, a diet history questionnaire, gastrointestinal symptoms questionnaires,
					perceived stress questionnaires, physical activity questionnaires, and sleep questionnaires. We have also begun the analysis of fecal samples from the Human Microbiome Project and the associated metadata. The integration of this data into a
					single knowledgebase of comparable samples using our optimized pipeline will provide the real value of our prototype.
					<br><br><br><br>
					<b>Objective</b>
					<br>
					Gut feeling knowledgebase is a reference database of healthy human gut microbiome. It is generated by a metagenomic analysis pipeline describes in our paper (doi: <a href="https://doi.org/10.1371/journal.pone.0206484"
						target="_blank">10.1371/journal.pone.0206484</a>), and includes three tools which are integrated in the HIVE platform. 49 healthy samples sequenced at GWU and 49 healthy samples taken from The Human Microbiome Project were analysed to
					create GutFeelingKB.

					<br><br>
					<b>GFKB downloads</b>
					<br><br>
					<table style="font-size:13px;" cellpadding=5 width=100% border=1>
						<tr>
							<th>Version</th>
							<th>Content files</th>
							<th>Format</th>
							<th>File size</th>
							<th>Release notes in plain text file</th>
							<th>Date Created</th>
						</tr>

						<tr>
							<td>v1.0</td>
							<td>
							  <a class=blue13 href="HTMLROOT/content/RLDA_KI_Analysis.pdf" download="RLDA_KI_Analysis.pdf">RLDA KI Analysis</a>
							
							</td>
							<td>pdf</td>
							<td>393KB</td>
							<td>
								<a>N/A</a>
							</td>
							<td>May 14 2021</td>
						</tr>
						
						<tr>
							<td>v1.0</td>
							<td>
							  <a class=blue13 href="HTMLROOT/content/ML_Matlab_Tutorial.pdf" download="ML_Matlab_Tutorial.pdf">ML Matlab Tutorial</a>
							
							</td>
							<td>pdf</td>
							<td>5.2MB</td>
							<td>
								<a>N/A</a>
							</td>
							<td>January 6 2021</td>
						</tr>
						
						<tr>
							<td>v4.0</td>
							<td>
								<a class=blue13 href="HTMLROOT/content/GutFeelingKnowledgeBase-v4-Master_List.csv" download="GutFeelingKnowledgeBase-v4-Master_List.csv">GutFeelingKnowledgeBase-v4-Master_List.csv</a>
								<br>
								<a class=blue13 href="HTMLROOT/content/GutFeelingKnowledgeBase-v4-Epilepsy_Data.csv" download="GutFeelingKnowledgeBase-v4-Epilepsy_Data.csv">GutFeelingKnowledgeBase-v4-Epilepsy_Data.csv</a>
							</td>
							<td>csv</td>
							<td>290KB
								<br>99KB</td>
							<td>
								<a class=blue13 href="BASEURL/gfkb/GutFeelingKnowledgeBaseNotesV4">GutFeeling Knowledge Base Notes v4.0</a>
							</td>
							<td>March 31 2020</td>
						</tr>

						<tr>
							<td>v3.0</td>
							<td>
								<a class=blue13 href="HTMLROOT/content/GutFeelingKnowledgeBase-v3.csv" download="GutFeelingKnowledgeBase-v3.csv">GutFeelingKnowledgeBase-v3.csv</a>
							</td>
							<td>csv</td>
							<td>44K</td>
							<td>
								<a class=blue13 href="BASEURL/gfkb/GutFeelingKnowledgeBaseNotesV3">GutFeeling Knowledge Base Notes v3.0</a>
							</td>
							<td>Aug 30 2019</td>
						</tr>

						<tr>
							<td>v2.6</td>
							<td>
								<a class=blue13 href="HTMLROOT/content/GutFeelingKnowledgeBase-v2.6.csv" download="GutFeelingKnowledgeBase-v2.6.csv">GutFeelingKnowledgeBase-v2.6.csv</a>
							</td>
							<td>csv</td>
							<td>44K</td>
							<td>
								<a class=blue13 href="BASEURL/gfkb/GutFeelingKnowledgeBaseNotesV2_6">GutFeeling Knowledge Base Notes v2.6</a>
							</td>
							<td>Jul 23 2018</td>
						</tr>


						<tr>
							<td>v2.6</td>
							<td>
								<a class=blue13 href="HTMLROOT/content/HumanGutDB-v2.6.fasta" download="HumanGutDB-v2.6.fasta">HumanGutDB-v2.6.fasta</a>
							</td>
							<td>fasta</td>
							<td>549M</td>
							<td>
								<a class=blue13 href="BASEURL/gfkb/HumanGutDBv2_6">HumanGutDB v2.6 Notes</a>
							</td>
							<td>Jul 23 2018</td>
						</tr>

						<tr>
							<td>v2.0</td>
							<td>
								<a class=blue13 href="HTMLROOT/content/GutFeelingKnowledgeBase-v2.0.csv" download="GutFeelingKnowledgeBase-v2.0.csv">GutFeelingKnowledgeBase-v2.0.csv</a>
							</td>
							<td>csv</td>
							<td>248K</td>
							<td>
								<a class=blue13 href="BASEURL/gfkb/GutFeelingKnowledgeBaseNotesV2">GutFeeling Knowledge Base Notes v2.0</a>
							</td>
							<td>2017</td>
						</tr>

						<tr>
							<td>v2.0</td>
							<td>
								<a class=blue13 href="HTMLROOT/content/HumanGutDB-v2.0.fasta" download="HumanGutDB-v2.0.fasta">HumanGutDB-v2.0.fasta</a>
							</td>
							<td>fasta</td>
							<td>533M</td>
							<td>
								<a class=blue13 href="BASEURL/gfkb/HumanGutDBv2">HumanGutDB v2.0 Notes</a>
							</td>
							<td>2017</td>
						</tr>

						<tr>
							<td>v2.0</td>
							<td>
								<a class=blue13 href="HTMLROOT/content/blackList-v2.0.csv" download="blackList-v2.0.csv">blockList-v2.0.csv</a>
							</td>
							<td>csv</td>
							<td>16K</td>
							<td>
								<a class=blue13 href="BASEURL/gfkb/blackListNotesV2">Block List Notes v2.0</a>
							</td>
							<td>2017</td>
						</tr>

						<tr>
							<td>v2.0</td>
							<td>
								<a class=blue13 href="HTMLROOT/content/unalignedContigsGFKB-v2.0.fasta" download="unalignedContigsGFKB-v2.0.fasta">unalignedContigsGFKB-v2.0.fasta</a>
							</td>
							<td> fasta </td>
							<td>3.2G</td>
							<td>
								<a class=blue13 href="BASEURL/gfkb/unalignedContigsGFKBNotesV2">Unaligned Contigs GFKB Notes</a>
							</td>
							<td>2017</td>
						</tr>

					</table>
                                        <br><br>
					<h5>GutFeeling Knowledgebase Relative Abundance Profiles</h5>
					<h6 id="graph_level_title"></h6>
					<div id="doughnut_chart"></div>
					<div id="graph_buttons">
						<button type="button" id="species">Species Level</button>
						<button type="button" id="genus">Genus Level</button>
					</div>

					<br><br>
					<b> <a href="BASEURL/filterednt">Filtered nt</a></b>
					<br>
					<br>The Filtered NT dataset is generated by excluding sequences from the whole nucleotide file provided by NCBI, based on whether they have unwanted taxonomy names or any child taxonomy name of these unwanted ones.
					<br><br>
					<b>Metagenomics Pipeline</b>
					<br>
					We use a two-step pipeline for metagenomic analysis; CensuScope and Hexagon. CensuScope is a census-based tool that randomly samples a user-defined number of reads and BLASTs them against a reference DB.
					Our reference database (a filtered version of NTdb) is the NCBI Nucleotide db with all of the sequences lacking a clear taxonomic lineage filtered out. All artificial sequences have been removed either by our automated filter or manually,
					once an artificial sequence is identified during post analysis processing
					Sequences identified by CensuScope are used as references in Hexagon alignments. HIVE-hexagon, a K-mer based aligner, is more sensitive and faster than current standard alignment algorithms. HIVE-hexagon offers a decrease in computational
					cost, memory requirement and time for processing.



					<br><br><br><br>
					<b>Publications</b>
					<br>
					<p>Please cite use with one of the following:</p>
					<ol style="list-style-position:inside;">
						<li><a href="https://orcid.org/0000-0003-1409-4549
" target="_blank">King CH</a>, Desai H, Sylvetsky AC, <a href="http://orcid.org/0000-0001-6897-5419" target="_blank">LoTempio J</a>, Ayanyan S, Carrie J, <a href="http://orcid.org/0000-0002-0836-3389" target="_blank">Crandall K</a>, Fochtman B,
							Gasparyan L, Gulzar N, Howell P, Issa N, Krampis K, Mishra L, Morizono H, Pisegna JR, Rao S, <a href="http://orcid.org/0000-0001-6369-3703" target="_blank">Ren Y</a>, Simonyan V, Smith K, VedBrat S, Yao M, <a
								href="http://orcid.org/0000-0001-8823-9945" target="_blank">Mazumder R.</a> Baseline human gut microbiota profile in healthy people and standard reporting template. PLOS ONE 2019. doi: <a
								href="https://doi.org/10.1371/journal.pone.0206484" target="_blank">10.1371/journal.pone.0206484</a></li>
						<br><br>
						<li>Shamsaddini A, Pan Y, Johnson WE, Krampis K, Shcheglovitova M, Simonyan V, Zanne A, Mazumder R. Census-based rapid and accurate metagenome taxonomic profiling. BMC Genomics. 2014;15(1):918. PMID: <a
								href="http://www.ncbi.nlm.nih.gov/pubmed/25336203" target="_blank">25232094</a></li>
						<br><br>
						<li>Santana-Quintero L, Dingerdissen H, Thierry-Mieg J, Mazumder R, Simonyan V. HIVE-Hexagon: High-Performance, Parallelized Sequence Alignment for Next-Generation Sequencing Data Analysis. PLOS One. 2014;9(6):e99033. PMID: <a
								href="http://www.ncbi.nlm.nih.gov/pubmed/24918764" target="_blank">24918764</a>.</li>
						<br><br>
						<li>Simonyan V and Mazumder R. High-performance Integrated Virtual Environment (HIVE) Tools and Applications for Big Data Analysis. Genes, 2014 Sep 30;5(4): 957-981. PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25271953"
								target="_blank">25271953</a>.</li>
					</ol>
					<br><br>
					<b>Funding</b>
					<br>
					Current/past: NSF, Otsuka, MGPC

					<br><br>

					<b>Acknowledgements</b>
					<br>
					We would like to thank the following individuals for their significant work in curation and annotation of the GFKB:<br>
					Stephanie Singleton: <a href="https://orcid.org/0000-0003-0256-9834" target="_blank">https://orcid.org/0000-0003-0256-9834</a><br>
					Lindsay Hopson: <a href="https://orcid.org/0000-0002-1586-1693" target="_blank">https://orcid.org/0000-0002-1586-1693</a><br>
					Jiuge (April) Yang: <a href="https://orcid.org/0000-0002-6497-0714" target="_blank">https://orcid.org/0000-0002-6497-0714</a><br>
					James Ziegler: <a href="https://orcid.org/0000-0003-0098-855X" target="_blank">https://orcid.org/0000-0003-0098-855X</a><br>
					Noyanika Vattathara: <a href="https://orcid.org/0000-0003-0330-6307" target="_blank">https://orcid.org/0000-0003-0330-6307</a><br>
					Mark Moussa
					Daniel Zhang:  <a href="https://orcid.org/0000-0002-3672-0179" target="_blank">https://orcid.org/0000-0002-3672-0179</a><br>
					Vishya Adipudi:  <a href="https://orcid.org/0000-0003-0411-1368" target="_blank">https://orcid.org/0000-0003-0411-1368</a><br>
					Tyson Dawson: <a href="https://orcid.org/0000-0003-4888-9673" target="_blank">https://orcid.org/0000-0003-4888-9673</a><br>
					Cameron Sabet: <a href="https://orcid.org/0000-0003-2299-1426" target="_blank">https://orcid.org/0000-0003-2299-1426</a><br>
					Yukta Chidanandan: <a href="https://orcid.org/0000-0001-5703-5667" target="_blank">https://orcid.org/0000-0001-5703-5667</a><br>
					Valery Simonyan: <a href="https://orcid.org/0000-0002-2577-3240" target="_blank">https://orcid.org/0000-0002-2577-3240</a><br>
					Nicole Post: <a href="https://orcid.org/0000-0002-0457-7056" target="_blank">https://orcid.org/0000-0002-0457-7056</a><br>
					Ben Osborne: <a href="https://orcid.org/0000-0002-9007-8746" target="_blank">https://orcid.org/0000-0002-9007-8746</a><br>
					Sophie Halkett: <a href="https://orcid.org/0000-0001-9721-3181" target="_blank">https://orcid.org/0000-0001-9721-3181</a><br>
					Miguel Mazumder: <a href="https://orcid.org/0000-0003-1181-8118" target="_blank">https://orcid.org/0000-0003-1181-8118</a><br>
				</span>


			</td>
		</tr>
	</table>
	<br><br><br>
	Questions / Comments
	If you have any questions or comments regarding GutFeelingKB, please contact Raja Mazumder (mazumder@gwmail.gwu.edu).
  <script src="BASEURL/gfkb/jslib/d3.v5.min.js"></script>
  <script src="BASEURL/gfkb/jslib/graphScript.js"></script>
</body>
